solutions/swb-ui/README.md

```diff
@@ -14,23 +14,15 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ### Prerequisite:
 
-1. Make sure to Follow instructions in [here](../swb-reference/SETUP_v2p1.md##installation) to setup installation of API.
+1. Follow instructions [here](../swb-reference/SETUP_v2p1.md##installation) to setup installation of API.
```

Comment:
Nitpick: Since there's only one item here, we don't need a bulleted list.
```suggestion
Follow instructions [here](../swb-reference/SETUP_v2p1.md##installation) to setup installation of API.
```

Reply:
Fixed